ml-agents/mlagents/trainers/trajectory.py

```diff
@@ -18,48 +18,40 @@ class AgentExperience(NamedTuple):
     memory: np.ndarray
 
 
-class SplitObservations(NamedTuple):
-    vector_observations: np.ndarray
-    visual_observations: List[np.ndarray]
+class ObsUtil:
+    @staticmethod
+    def get_name_at(index: int) -> str:
+        """
+        returns the name of the observation given the index of the observation
+        """
+        return "obs_%d" % index
```

Comment:
How about about tuple instead of string concatenation? 
```
$ python3 -m timeit '{("obs", n): n for n in range(10000)}'
500 loops, best of 5: 718 usec per loop
$ python3 -m timeit '{f"obs_{n}": n for n in range(10000)}'
200 loops, best of 5: 1.77 msec per loop
$ python3 -m timeit '{f"obs_%d" % n: n for n in range(10000)}'
200 loops, best of 5: 1.84 msec per loop
```
If that doesn't convince you, at least use an f-string.

Reply:
I use an f-string now